// ccstiet/index.js
// ccstiet module

function hello() {
  return "\x1b[32mHello from CCS!\x1b[0m";
}

// Core 23-24
function abhinav() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mAbhinav Jain\x1b[0m";
}

function adrija() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mAdrija Sengupta\x1b[0m";
}

function akarsh() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mAkarsh Shrivastava\x1b[0m";
}

function armaan() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mArmaan Mittal\x1b[0m";
}

function dev() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mDev Goyal\x1b[0m";
}

function dinesh() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mDinesh Agrawal\x1b[0m";
}

function gopal() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mGopal Agarwal\x1b[0m";
}

function gurman() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mGurman Kaur\x1b[0m";
}

function hushraj() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mHushraj Singh\x1b[0m";
}

function jahanvi() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mJahanvi Srivastava\x1b[0m";
}

function jasdeep() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mJasdeep Singh\x1b[0m";
}

function kanishk() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mKanishk Jagya\x1b[0m";
}

function pancham() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mPancham Agarwal\x1b[0m";
}

function pariansh() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mPariansh Mahajan\x1b[0m";
}

function reyan() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mReyan Singh\x1b[0m";
}

function saaransh() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mSaaransh Gupta\x1b[0m";
}

function sharath() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mSharath Chandra Chinnarigari\x1b[0m";
}

function shaurya() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mShaurya Verma\x1b[0m";
}

function sparsh() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mSparsh Rastogi\x1b[0m";
}

function swapnil() {
  return "\x1b[32mHello Core 23-24:\x1b[0m \x1b[31mSwapnil Chhibber\x1b[0m";
}

// Exbo 23-24
function aditya() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mAditya Parmar\x1b[0m";
}

function harsiddak() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mHarsiddak Singh Bedi\x1b[0m";
}

function garv() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mGarv Chandalia\x1b[0m";
}

function ansh() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mAnsh Bajaj\x1b[0m";
}

function chandravo() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mChandravo Bhattacharya\x1b[0m";
}

function siddharth() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mSiddharth Banga\x1b[0m";
}

function tijil() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mTijil Malhotra\x1b[0m";
}

function vivaan() {
  return "\x1b[32mHello Exbo 23-24:\x1b[0m \x1b[31mVivaan Taneja\x1b[0m";
}

module.exports = {
  hello,
  abhinav,
  adrija,
  akarsh,
  armaan,
  dev,
  dinesh,
  gopal,
  gurman,
  hushraj,
  jahanvi,
  jasdeep,
  kanishk,
  pancham,
  pariansh,
  reyan,
  saaransh,
  sharath,
  shaurya,
  sparsh,
  swapnil,
  aditya,
  harsiddak,
  garv,
  ansh,
  chandravo,
  siddharth,
  tijil,
  vivaan,
};
